man/systemd.service.xml

```diff
@@ -153,77 +153,89 @@
+          <option>exec</option>, <option>forking</option>, <option>oneshot</option>, <option>dbus</option>,
+          <option>notify</option> or <option>idle</option>:</para>
+
+          <itemizedlist>
+            <listitem><para>If set to <option>simple</option> (the default if neither <varname>Type=</varname> nor
+            <varname>BusName=</varname>, but <varname>ExecStart=</varname> are specified), it is expected that the
+            process configured with <varname>ExecStart=</varname> is the main process of the service. In this mode, if
+            the process offers functionality to other processes on the system, its communication channels should be
+            installed before the service is started up (e.g.  sockets set up by systemd, via socket activation), as the
+            service manager will immediately proceed starting follow-up units, right after creating the main service
+            process, and before executing the service's binary.</para></listitem>
+
+            <listitem><para>The <option>exec</option> type is similar to <option>simple</option>, but the service
+            manager will delay starting of follow-up units until not only the service's main process has been created
+            but also the main service binary has been executed. (Or in other words: <option>simple</option> proceeds
+            with further jobs right after <function>fork()</function> returns, while <option>exec</option> will not
+            proceed before both <function>fork()</function> and <function>execve()</function> in the service process
+            succeeded.) Note that this means <command>systemctl start</command> command lines for
+            <option>simple</option> services will report success even if the service's binary cannot be invoked
+            successfully (for example because the selected <varname>User=</varname> doesn't exist, or the service
+            binary is missing), while the same command line for <option>exec</option> services will report failure in
+            such cases.</para></listitem>
+
+            <listitem><para>If set to <option>forking</option>, it is expected that the process configured with
+            <varname>ExecStart=</varname> will call <function>fork()</function> as part of its start-up. The parent
+            process is expected to exit when start-up is complete and all communication channels are set up. The child
+            continues to run as the main service process. This is the behavior of traditional UNIX services. If this
+            setting is used, it is recommended to also use the <varname>PIDFile=</varname> option, so that systemd can
+            identify the main process of the service. systemd will proceed with starting follow-up units as soon as the
+            parent process exits.</para></listitem>
+
+            <listitem><para>Behavior of <option>oneshot</option> is similar to <option>simple</option>; however, it is
```

Comment:
"it is expected" seems misplaced. Maybe "systemd will consider the unit started after the process exits. It will then start follow-up units."